scripts/release/upload_assets.sh

```diff
@@ -31,14 +31,10 @@ MODULE_VERSION=${PULL_BASE_REF} make build-manifests
 echo "Generated template-operator.yaml:"
 cat template-operator.yaml
 
-MODULE_VERSION=${PULL_BASE_REF} make build-module
```

Comment:
Why do we need to get rid of module template generation?

Reply:
moduletemplate is only for managed use case, which is generated by internal pipeline into kcp, the open source release, we don't need to provide it, you can check other kyma module release as example. https://github.com/kyma-project/keda-manager/releases/tag/1.2.0

Reply:
Alright, LGTM :D